fe/lib/components/BrandSelect/PaginatedBrands.tsx

```diff
@@ -36,11 +36,11 @@ export const PaginatedBrands = ({
     return (
       <Notice tone="info">
         <Text>
-          You have no brands. Create a new one in the{' '}
+          You have no brands. Upload a new brand on the{' '}
           <SmartTextLink href="https://talent.seek.com.au/account/branding">
```

Comment:
This hyperlink could present localisation challenges. Should we remove it?

Reply:
That's an interesting point! I'd leave it in for now but something to think about - e.g. I wonder if we should have a URL redirection service for these long-lived URLs that partners might incorporate in to their software?